recipes/gamenetworkingsockets/all/conandata.yml

```diff
@@ -1,10 +1,4 @@
 sources:
-  "1.3.0":
-    url: "https://github.com/ValveSoftware/GameNetworkingSockets/archive/refs/tags/v1.3.0.zip"
-    sha256: "22e409546babc449c44f492b253b547a2f5f11abe11a100686a10a990b5091cd"
-patches:
-  "1.3.0":
-    - patch_file: "patches/001-disable-runtime-override.patch"
-      base_path: "source_subfolder"
-    - patch_file: "patches/002-either-static-or-shared.patch"
-      base_path: "source_subfolder"
+  "1.4.1":
```

Comment:
Can we keep 1.3 for ease of migration?

Reply:
I'm not looking forward to restoring support for it. This recipe was a pretty stubborn one. Plus, I've forgotten most of the details by now...